### ISSUE TITLE
ripple-0.16.8.js error: fails due to the ripple-lib missing arguments in...

### ISSUE THREAD

**Issue by jnr101** [ripple/ripple-lib#702]
Hello,

When building the ripple-0.16.8.js following the guidelines from [https://ripple.com/build/rippleapi-beginners-guide/#build-instructions](url) an error occurs when using the ripple-0.16.8.js.

it fails due to the ripple-lib missing arguments in the function(modules, exports) on line 5500 where:      module.exports = _;

Can you check please?

Thanks
Jn_r
##

**Comment by whotooktwarden** [comment]
I can confirm this issue.  

https://xagate.com/rippleapi/get_wallet.html uses 0.16.5.js meanwhile I have copied the file to https://xagate.com/rippleapi/testbug.html modifying the source to 0.16.8.js.  The latter produces the following errors:

ripple-0.16.8.js:5500 Uncaught ReferenceError: _ is not defined and undefined

testbug.html:7 Uncaught TypeError: Cannot read property 'RippleAPI' of undefined

**Comment by clark800** [comment]
@darkdarkdragon made lodash a separate dependency, try including lodash 3.1 separately

**Comment by intelliot** [comment]
@jnr101 This shouldn't be a problem in the current version of ripple-lib. Please let us know if you find otherwise. Thanks!

**Comment by jnr101** [comment]
I tested it and it worked. Thanks!
Maybe you could update the 'master' to latest version? Following instructions from [https://ripple.com/build/rippleapi-beginners-guide/#build-instructions](https://ripple.com/build/rippleapi-beginners-guide/#build-instructions) I ended up with version 0.17.6 and had to do a "git checkout develop" to get version 0.17.7

**Comment by intelliot** [comment]
@jnr101 Good catch. I've updated `master` to 0.17.7. I also took a closer look at this issue. I think the problem was that lodash needs to be included on your page, e.g.
```
  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.15.0/lodash.js"></script>
```
It works now because our example code now includes this script tag.

By the way, we plan to release 0.17.8 shortly, which works around a bug that can cause Safari 10.1 to freeze. I recommend upgrading when it's available next week :)
